docs/manual/02_Quick_start.md

```diff
@@ -0,0 +1,151 @@
+
+### Verify
+
+To verify that Clusterous is installed, try:
+
+    clusterous --help
+    
+And you should see Clusterous' help output. Like a typical Unix command line tool, it should show you the list of commands and options.
+
+## Configuring
+Clusterous creates clusters on your own AWS account, so before you start using Clusterous, you need provide your AWS credentials and run the interactive Clusterous setup wizard.
+
+The AWS keys are in the form of an Access Key ID and Secret Access Key, and are provided to you by AWS when you create an IAM user in your AWS account. You may refer to this manual's guide [TODO: LINK] to preparing your AWS account for use. Your AWS credentials will typically look something like:
+
+    Access Key ID: AKIAIOSFODNN7EXAMPLE
+    Secret Access Key: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+
+Once you have obtained your AWS keys, run the `setup` command to launch the interactive wizard:
+
+    clusterous setup
+
+The setup wizard will start by asking you to enter your AWS keys (which you can copy/paste into the terminal). Following that, it will guide your through setting up some AWS resources that Clusterous needs for launching and managing clusters. In each case, Clusterous will give you the option of either picking an existing one or creating a new one. In general, if you expect to share your clusters with collaborators, it is recommended that you use the same resources as them. If you are the first to use Clusterous on your account, or will be working alone, feel free to create your own resources using the setup wizard. To quit the setup wizard at any time, enter `Ctrl-C`.
+
+Some of the questions the setup wizard will ask for:
+
+- The AWS region you want to use. This corresponds to the location of their data centers, and you would typicaly choose the one geographically nearest to you.
+- The VPC (Virtual Private Cloud) to use, which pertains to AWS networking. If you are the first to use AWS on your account, feel free to create a new one. If a colleague is already using Clusterous on your account, it is best to choose the same one as them
+- A Key Pair, which is an SSH key for establishing a secure connection to your cluster. Clusterous needs a `.pem` key file to create and manage clusters. Like with VPCs, it is best to use an existing Key Pair if you are collaborating with others on your account. If chosing an existing Key Pair, Clusterous will ask you for the location of your key file.
+- An S3 bucket for storing Docker images. Like with other resources, you are best off sharing the S3 bucket with collaborators. Note that S3 bucket names are global across AWS, so people typically prefix their organisation name.
+- A name for this configuration profile. You need only one profile to begin with. Clusterous lets you have multiple sets of configuration, which is useful if you run on multiple regions or AWS accounts.
+
+Once you have succesfully run the setup wizard, you are ready to start using Clusterous.
```

Comment:
It's a subtle difference, but I want to emphasise that the setup is required before you can do anything with Clusterous, not just creating a cluster.